Program.cs

```diff
@@ -1,7 +1,13 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+Console.Write ("Enter input: ");
+int n = int.Parse (Console.ReadLine ());
+int a = 0, b = 1, c, i = 2;

```

Comment:
Variable 'c' is not used anywhere outside the while loop ([Block Level Scope](https://www.geeksforgeeks.org/scope-of-variables-in-c-sharp/)). Can you declare/define the variable just inside of the loop?